### PR TITLE
feat: add `Series.plot.hist()` method

### PR DESCRIPTION
**Pull request by chelsea-lin**
Fixes internal issue b/322178004🦕

### CODE REVIEW

File: bigframes/series.py [hunk 1]
```diff
@@ -1524,6 +1525,49 @@ def sample(
             self._block._split(ns=ns, fracs=fracs, random_state=random_state)[0]
         )
 
+    def hist(
```
Comment (tswast) [inline bigframes/series.py:1528]: Can we expose this as `Series.plot.hist`, instead? https://pandas.pydata.org/pandas-docs/version/2.1/reference/api/pandas.Series.plot.hist.html

I'm not 100% sure why pandas provides Series.plot.hist and Series.hist https://pandas.pydata.org/pandas-docs/version/2.1/reference/api/pandas.Series.hist.html but  I think Series.plot.hist is the newer method and will make our docs organization easier.

Comment (chelsea-lin) [reply bigframes/series.py:1528]: That was what I confused too! Thanks for your suggestions!

Comment (chelsea-lin) [reply bigframes/series.py]: Updated it into `Series.plot.hist`. Please review it again!

**Review by orrbradford — COMMENTED** [review]
LGTM! I'll to @tswast to approve

**Comment by chelsea-lin** [comment]
We don't need this change anymore with the change: https://github.com/googleapis/python-bigquery-dataframes/pull/420

**Pull request closed without merging**